### ISSUE TITLE
Add group selection

### ISSUE THREAD

**Issue by zecakeh** [etesync/etesync-notes#113]
To be able to do batch actions on notes: delete, share, export, move…

**Comment by tasn** [comment]
That could be cool indeed! Though to be honest, there are other issues I'm more excited about, as doing actions on multiple notes is probably quite rare compared to other tasks.

**Comment by zecakeh** [comment]
It's not rare to have to batch delete when you're working on importing or creating notes 😛

**Comment by tasn** [comment]
Yeah, but not for the general population who doesn't do these things every day. :)
